Gemfile.lock

```diff
@@ -1,67 +1,67 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    actioncable (6.0.2.1)
-      actionpack (= 6.0.2.1)
+    actioncable (6.0.2.2)
+      actionpack (= 6.0.2.2)
       nio4r (~> 2.0)
       websocket-driver (>= 0.6.1)
-    actionmailbox (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activejob (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
-      activestorage (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    actionmailbox (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activejob (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
+      activestorage (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       mail (>= 2.7.1)
-    actionmailer (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      actionview (= 6.0.2.1)
-      activejob (= 6.0.2.1)
+    actionmailer (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      actionview (= 6.0.2.2)
+      activejob (= 6.0.2.2)
       mail (~> 2.5, >= 2.5.4)
       rails-dom-testing (~> 2.0)
-    actionpack (6.0.2.1)
-      actionview (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    actionpack (6.0.2.2)
+      actionview (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       rack (~> 2.0, >= 2.0.8)
       rack-test (>= 0.6.3)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.0, >= 1.2.0)
-    actiontext (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
-      activestorage (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
+    actiontext (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
+      activestorage (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
       nokogiri (>= 1.8.5)
-    actionview (6.0.2.1)
-      activesupport (= 6.0.2.1)
+    actionview (6.0.2.2)
+      activesupport (= 6.0.2.2)
       builder (~> 3.1)
       erubi (~> 1.4)
       rails-dom-testing (~> 2.0)
       rails-html-sanitizer (~> 1.1, >= 1.2.0)
-    activejob (6.0.2.1)
-      activesupport (= 6.0.2.1)
+    activejob (6.0.2.2)
+      activesupport (= 6.0.2.2)
       globalid (>= 0.3.6)
-    activemodel (6.0.2.1)
-      activesupport (= 6.0.2.1)
-    activerecord (6.0.2.1)
-      activemodel (= 6.0.2.1)
-      activesupport (= 6.0.2.1)
-    activestorage (6.0.2.1)
-      actionpack (= 6.0.2.1)
-      activejob (= 6.0.2.1)
-      activerecord (= 6.0.2.1)
+    activemodel (6.0.2.2)
+      activesupport (= 6.0.2.2)
+    activerecord (6.0.2.2)
+      activemodel (= 6.0.2.2)
+      activesupport (= 6.0.2.2)
+    activestorage (6.0.2.2)
+      actionpack (= 6.0.2.2)
+      activejob (= 6.0.2.2)
+      activerecord (= 6.0.2.2)
       marcel (~> 0.3.1)
-    activesupport (6.0.2.1)
+    activesupport (6.0.2.2)
       concurrent-ruby (~> 1.0, >= 1.0.2)
       i18n (>= 0.7, < 2)
       minitest (~> 5.1)
       tzinfo (~> 1.1)
       zeitwerk (~> 2.2)
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
-    autoprefixer-rails (9.7.4)
+    autoprefixer-rails (9.7.5)
       execjs
     bindex (0.8.1)
-    bootsnap (1.4.5)
+    bootsnap (1.4.6)
```

Comment:
### [bootsnap](https://github.com/Shopify/bootsnap)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.4.5 → 1.4.6

[_[change-log](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md), [source-code](https://github.com/Shopify/bootsnap)_]





<details>
<summary>Commits</summary>

A change of **17** commits. See the full changes on [the compare page](https://github.com/Shopify/bootsnap/compare/v1.4.5...v1.4.6).

These are the first 10 commits:
- (f15ce20) [Remove a temporary variable](https://github.com/Shopify/bootsnap/commit/f15ce208ce2aa911e100f1fb9658b2e6f586920a)
- (826c0d6) [Add const qualifiers to the parameters and remove unsafe poi…](https://github.com/Shopify/bootsnap/commit/826c0d61dbd60db62f604b97cb9915656ecc4567)
- (2a3644c) [Only strip elidable extensions to make short forms](https://github.com/Shopify/bootsnap/commit/2a3644ce35a325c8427d7079c1e19080fe0596d5)
- (065f909) [Merge pull request #274 from Ryooooooga/remove-unsafe-pointe…](https://github.com/Shopify/bootsnap/commit/065f909a0e0fa05ccc4e26d5ede3bcc655779020)
- (ecdd65e) [Only strip elidable extensions to make short forms (#271)](https://github.com/Shopify/bootsnap/commit/ecdd65ea370024ad855bdfb5b6a096b48783c343)
- (b720530) [Update minimum ruby version and fix rubocop](https://github.com/Shopify/bootsnap/commit/b7205305841a792064ffcc4e4a5e127ff688ee37)
- (78be2f5) [Merge pull request #280 from alebruck/update-ruby-version](https://github.com/Shopify/bootsnap/commit/78be2f5aec196d8d4a72f21ba03ad8fd41adf276)
- (8d4b28a) [Don't break when Pathname is passed to LoadedFeaturesIndex](https://github.com/Shopify/bootsnap/commit/8d4b28ac609ddb0c613e41bba0cd428e591cc69f)
- (9278a85) [Don't break when Pathname is passed to LoadedFeaturesIndex (…](https://github.com/Shopify/bootsnap/commit/9278a85ffba5590c782ec7f5577064ec689b8ddf)
- (70bef22) [Update documentation on os_version -> ruby_platform](https://github.com/Shopify/bootsnap/commit/70bef22bc0b86436384afe6340ce8a68b9e3cd0f)


</details>